huaweicloud/services/elb/resource_huaweicloud_elb_logtanks.go

```diff
@@ -0,0 +1,144 @@
+package elb
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/elb/v3/logtanks"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func ResourceLogTanksV3() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceLogTanksV3Create,
+		ReadContext:   resourceLogTanksV3Read,
+		UpdateContext: resourceLogTanksV3Update,
+		DeleteContext: resourceLogTanksV3Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"loadbalancer_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"log_group_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"log_topic_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceLogTanksV3Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	elbClient, err := config.ElbV3Client(config.GetRegion(d))
+	if err != nil {
+		return diag.Errorf("error creating elb client: %s", err)
```

Comment:
s/elb/ELB v3
ELB is an abbreviation.

Reply:
All other files use this format, maybe it's better to keep consistent here.

Reply:
uppercase for service name is better.

Reply:
done